### PR TITLE
Settlement-v2 merge into master

### PR DESCRIPTION
**Pull request by ggrg**
Probably, it is good time to consider merging into master to avoid further maintenance

### CODE REVIEW

File: migrations/910102_hotfix954.js [hunk 1]
```diff
@@ -25,33 +25,7 @@
...
- - Name Surname <name.surname@gatesfoundation.com>
-
- * Miguel de Barros <miguel.debarros@modusbox.com>
- --------------
- ******/
-
-'use strict'
-
-exports.up = async (knex) => {
+exports.up = async (knex, Promise) => {
```
Comment (rmothilal) [inline migrations/910102_hotfix954.js]: Why did you remove the license text?

File: seeds/z1000_settlementModel.js- [hunk 2]
```diff
@@ -0,0 +1,56 @@
+/*****
```
Comment (rmothilal) [inline seeds/z1000_settlementModel.js-]: why does the filename have a dash at the end of it?

Comment (ggrg) [reply seeds/z1000_settlementModel.js-]: It is currently not executed, but I just wanted to preserve it for a while until it's absolutely certain, there won't be any generic settlement models to be seeded. Please, let me know if it would be better to delete the script.

Comment (rmothilal) [reply seeds/z1000_settlementModel.js-]: I would make a note in the file if you wish to keep it

File: migrations/910102_hotfix954.js (hunk 1 above)
Comment (rmothilal) [reply migrations/910102_hotfix954.js]: aha thanks

File: seeds/z1000_settlementModel.js- (hunk 2 above)
Comment (rmothilal) [reply seeds/z1000_settlementModel.js-]: I would suggest making a Note in the file stating why it is there and that it is not used. also consider appending or prepending the file name with ignore/deprecated or an appropriate term
